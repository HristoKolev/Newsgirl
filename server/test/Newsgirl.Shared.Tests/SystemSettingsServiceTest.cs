using System.Threading.Tasks;
using Newsgirl.Testing;
using Xunit;

namespace Newsgirl.Shared.Tests
{
    public class SystemSettingsServiceTest : DatabaseTest
    {
        [Fact]
        public async Task ReadSettings_Returns_Correct_Result()
        {
            var systemSettingsService = new SystemSettingsService(this.Db);

            var settings = await systemSettingsService.ReadSettings<SystemSettingsModel>();
            
            Snapshot.Match(settings);
        }
    }
}
