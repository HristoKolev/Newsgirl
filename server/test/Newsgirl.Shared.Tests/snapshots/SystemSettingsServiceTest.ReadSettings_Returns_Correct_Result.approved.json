{
  "FetcherCyclePause": 0,
  "HttpClientRequestTimeout": 120,
  "HttpClientUserAgent": "Mozilla/5.0 (X11; Linux x86_64) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/72.0.3626.81 Safari/537.36",
  "ParallelFeedFetching": true
}